apps/users-service/types/apr-intercept.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'apr-intercept';
```

Comment:
We should create proper definitions for these so that we don't have `any`s floating around in our code. No need to worry about that now though, I can see if I can come up with good definitions for these packages once it's merged into master.